html2asketch/document.js

```diff
@@ -34,7 +36,7 @@ class Document {
   }
 
   addTextStyle(textLayer) {
-    this._textStyles.push(textStyleToSharedStyle(textLayer));
+    this._textStyles.push(textStyleToSharedStyle(textLayer, this._objectID, this._textStyles.length));
```

Comment:
depending on the order here isn't perfect - maybe we could use names instead?

Reply:
probably, but what is the name of text style? where does it come from?

Reply:
text styles are added manually:

https://github.com/brainly/html-sketchapp-style-guide/blob/master/src/styleguide2asketch.js#L115-L116

and to work correctly, they need unique names